src/main/kotlin/frc/robot/subsystems/vision/PhotonVisionIOReal.kt

```diff
@@ -24,28 +25,32 @@ class PhotonVisionIOReal(private val camera: PhotonCamera, private val robotToCa
     }
 
     override fun updateInputs() {
-        val latestResult = camera.latestResult
+        val unreadResults = camera.allUnreadResults
 
-        if (!latestResult.hasTargets()) {
-            return
-        }
+        lateinit var estimatedPose: Optional<EstimatedRobotPose>
```

Comment:
I don't really understand why we need this to be `lateinit` can't this just be a val?

Reply:
I did it to fix something and than I fixed a different thing that rendered it unnecessary

Reply:
Then remove maybe revert it before resolving? @Emma03L

Reply:
Reverting the whole commit won't do any good